tests/isolated/rust-nightly.vader

```diff
@@ -0,0 +1,25 @@
+Include: ../include/setup.vader
+
+Execute (Setup: monkeypatch autoload/neomake/clippy.vim):
+  " Monkeypatch to force enable rust nightly
+  Save g:neomake_clippy_rustup_has_nightly
+  let g:neomake_clippy_rustup_has_nightly = 1
+
+  runtime autoload/neomake/makers/clippy.vim
```

Comment:
This does not test anything..

You should be able to just `Save g:neomake_clippy_rustup_has_nightly` and `let g:neomake_clippy_rustup_has_nightly` in a test (`Execute` block) that needs a particular mode for "has nightly".

Reply:
The above comment was under the assumption that the global would be used always.
It works / is needed for the previous/current code.